packages/reports/addon/templates/components/navi-column-config/metric.hbs

```diff
@@ -7,7 +7,8 @@
       @triggerId="{{this.classId}}-columnParameter" 
       @dropdownClass="navi-column-config-metric__parameter-dropdown"
       @selected={{this.currentParameter}}
-      @searchPlaceholder="Type to search"
+      @searchField="id"
```

Comment:
Why `id over `description`?

Reply:
We can change it to be description in both places, although id's being unique makes them nice for searching
https://github.com/yahoo/navi/blob/950c05b03d3d05a8c64ca8eee56cde79148bc852/packages/reports/addon/templates/components/metric-config.hbs#L18

Reply:
I agree about unique ID values. Just like most dimensions, users like to search by nice names. I don't think users will not really care about the param ID values.